### ISSUE TITLE
Include ""Strict-Transport-Security" headers in the Allowed_Headers

### ISSUE THREAD

**Issue by goutamnayak1706** [mendix/cf-mendix-buildpack#544]
Hi , 

By reading the documentation we found out that, Mendix supports the activation of HTTP headers by passing the env variable HTTP_RESPONSE_HEADERS , for ex
```
HTTP_RESPONSE_HEADERS: '{"Strict-Transport-Security":""max-age=31536000; includeSubDomains}'
```
But we find that the CF-mendix build pack does not have that in the list of allowed headers ( file: buildpack/core/nging.py )

```
ALLOWED_HEADERS = {
    "X-Frame-Options": r"(?i)(^allow-from https?://([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9])(\.([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9]))*(:\d+)?$|^deny$|^sameorigin$)",  # noqa: E501
    "Referrer-Policy": r"(?i)(^no-referrer$|^no-referrer-when-downgrade$|^origin|origin-when-cross-origin$|^same-origin|strict-origin$|^strict-origin-when-cross-origin$|^unsafe-url$)",  # noqa: E501
    "Access-Control-Allow-Origin": r"(?i)(^\*$|^null$|^https?://([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9])(\.([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9]))*(:\d+)?$)",  # noqa: E501
    "X-Content-Type-Options": r"(?i)(^nosniff$)",
    "Content-Security-Policy": r"[a-zA-Z0-9:;/''\"\*_\- \.\n?=%&+]+",
    "X-Permitted-Cross-Domain-Policies": r"(?i)(^all$|^none$|^master-only$|^by-content-type$|^by-ftp-filename$)",  # noqa: E501
    "X-XSS-Protection": r"(?i)(^0$|^1$|^1; mode=block$|^1; report=https?://([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9])(\.([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]{0,61}[a-zA-Z0-9]))*(:\d+)?$)",  # noqa: E501
}
```

Can we have "Strict-Transport-Security":""max-age=31536000; includeSubDomains}' also added to the list of allowed headers so that we can add it to our env variable and make it work.

**Comment by markbastiaans** [comment]
Hi @goutamnayak1706 - thanks for reporting this. This seems to be addressed in #545.

I'll close this issue if / when this PR is merged.

**Comment by markbastiaans** [comment]
PR is merged and will be in the next release
